content/src/main/content/jcr_root/apps/acs-commons/widgets/js.txt

```diff
@@ -14,4 +14,6 @@ validate_responsive_column_control_dialog.js
 add_dynamic_options_to_feed_importer.js
```

Comment:
This clientlib should be removed as it aggregates unrelated features. Rather each feature should be encapsulated in its own clientlib.

Reply:
@kwin youre saying create individual clientlibs for each logical set of functionality across these files? 

![image](https://user-images.githubusercontent.com/1451868/52737642-d7975900-2f9a-11e9-9437-a0d8dc31bcb8.png)

Same with w/ the ClassicUI widgets.

I tend to agree. You have any time to define the CL's?

Reply:
Unfortunately not in the near future. But IMHO it is more important to fix the one for Touch UI though, as Classic is deprecated anyhow...